client/src/components/VideoCard.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { AspectRatio, Flex, Box, Heading, IconButton } from "@chakra-ui/react";
+import './VideoCard.css';
+import { FaThumbsUp, FaThumbsDown, FaBan } from 'react-icons/fa';
+
+const VideoCard = ({ videoData, videosArr, setVideosArr }) => {
+	const [ratingUp, setRatingUp] = useState(false);
+	const [ratingDown, setRatingDown] = useState(false);
+	const [ratingNr, setRatingNr] = useState(videoData.rating);
+
+	const incrementRating = () => {
```

Comment:
what will happen if I click the first decrement and then increment and decrement again?